.github/workflows/ci.yml

```diff
@@ -1,44 +1,45 @@
 name: CI
-
-on:
+'on':
```

Comment:
Formatting has been mangled on a few things from the looks of it, same with `jobs.strategy.matrix.os` and `jobs.strategy.matrix.node-version`